css/properties/mask-image.json

```diff
@@ -0,0 +1,184 @@
+{
+  "css": {
+    "properties": {
+      "mask-image": {
```

Comment:
I think most of the comments on the current BCD should be added.
[1] Implemented in version 8.0, Chrome initially only supported the -webkit prefixed versions of gradients as values. Later, support for the unprefixed versions was added.
[2] Implemented in version 4.0, Safari initially only supported the -webkit prefixed versions of gradients as values.
[3] Android initially only supported the -webkit prefixed versions of gradients as values.
[4] iOS Safari initially only supported the -webkit prefixed versions of gradients as values

Reply:
OK, I think I initially misinterpreted these notes as meaning that the browsers once only supported prefixed `mask-image` then gained unprefixed support. But now I don't think that I understand the notes at all. What does "prefixed versions of gradients as values" mean, exactly? Something like `-webkit-linear-gradient()`?

Reply:
Yes, only -webkit-linear-gradient() functional was supported (and similar), not linear-gradient().